src/shared/hooks/useLaunchDarkly.ts

```diff
@@ -0,0 +1,56 @@
+import { LDFlagValue, useFlags, useLDClient } from 'launchdarkly-react-client-sdk';
+
+import { FeatureSegments, FeatureFlags, FeatureFlagsKeys } from 'shared/types/featureFlags';
+
+const omitKeys = ['firstName', 'lastName', 'email'];
```

Comment:
What do you think about calling this constant `PROHIBITED_PII_KEYS` to make its purpose very explicit? Also, perhaps moving it into `useLaunchDarkly.const.ts`?